apps/omg_child_chain/config/dev.exs

```diff
@@ -2,5 +2,4 @@
 use Mix.Config
 
 config :omg_child_chain,
-  ethereum_events_check_interval_ms: 500,
```

Comment:
:( missed this going through. In `dev` one often does 1-second blocks, so this was indeed useful

Reply:
where are you reading ethereum_events_check_interval_ms from the childchain application @pdobacz ?

Reply:
ahaaaaa, nice, :man_facepalming: